notebooks/community/ml_ops/stage7/model_monitoring_custom.ipynb

```diff
@@ -0,0 +1,1267 @@
+{
```

Comment:
<p>Can we add the gcloud commands to get the required permissions here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/890/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>This was a cut/paste from Marc's notebook, I will work on finding the gcloud syntax,</p>